src/main/java/uk/gov/hmcts/reform/security/keyvault/KeyVaultKeyStore.java

```diff
@@ -103,7 +111,7 @@ private Key getSmsTransportKey() {
     public Certificate engineGetCertificate(final String alias) {
         CertificateBundle certificateBundle = vaultService.getCertificateByAlias(alias);
         if (certificateBundle == null) {
-            return null;
+            certificateBundle = vaultService.getCertificateByAlias("test");
```

Comment:
maybe add a comment to let the world know why we are hacking this?

Reply:
```suggestion
            // AM will throw exceptions if it expects a certificate and the provider does not provide one
            // "test" is an RSA cert. Requested Cert will be unavailable for signing.
            certificateBundle = vaultService.getCertificateByAlias("test");
```